programs/server/config.xml

```diff
@@ -563,6 +563,31 @@
         <flush_interval_milliseconds>60000</flush_interval_milliseconds>
     </asynchronous_metric_log>
 
+    <!--
+        OpenTelemetry log contains OpenTelemetry trace spans.
+    -->
+    <opentelemetry_log>
+        <!--
+            The default table creation code is insufficient, this <engine> spec
+            is a workaround. There is no 'event_time' for this log, but two times,
+            start and finish. It is sorted by finish time, to avoid inserting
+            data too far away in the past (probably we can sometimes insert a span
+            that is seconds earlier than the last span in the table, due to a race
+            between several spans inserted in parallel). This gives the spans a
+            global order that we can use to e.g. retry insertion into some external
+            system.
+        -->
+        <engine>
+            engine MergeTree
+            partition by toYYYYMM(finish_date)
+            order by (finish_date, finish_time, trace_id)
```

Comment:
Can we remove finish_date? It makes queries a bit harder to type as usually you need to provide both date and time. Why not keep just finish_time?

Reply:
A separate date column is a way to save I/O for filters like `event_date = today()` -- the time column can be 300 times bigger than date, and we can avoid reading it. We use it for all system logs. With OpenTelemetry log it may be less important -- you are probably going to store and explore it using your tracing system, not with ClickHouse SQL.